Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -631,4 +631,43 @@ final class AttributeTests: XCTestCase {
       """
     )
   }
+  
+  func testInvalidWhitespaceBetweenAtSignAndIdenfifierIsDiagnosed() {
+    assertParse(
+      """
+      @1️⃣ MyAttribute
+      func foo() {}
+      """,
+      diagnostics: [
+        DiagnosticSpec(
+          message: "extraneous whitespace after '@' is not permitted",
+          fixIts: ["remove whitespace"]
+        )
+      ],
+      fixedSource: """
+      @MyAttribute
+      func foo() {}
+      """
+    )
+  }
+  
+  func testInvalidNewlineBetweenAtSignAndIdenfifierIsDiagnosed() {
+    assertParse(
+      """
+      @
+      MyAttribute
+      func foo() {}
+      """,
+      diagnostics: [
+        DiagnosticSpec(
+          message: "extraneous whitespace after '@' is not permitted",
+          fixIts: ["remove whitespace"]
+        )
```

Comment:
Did you forget to add a diagnostic marker `1️⃣` for this `DiagnosticSpec`? I would expect the test to fail as-is.